<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogIdentifierView</class>
 <widget class="QDialog" name="LogIdentifierView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logtexterkennung konfigurieren</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>631</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChangeIcon">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>170</y>
     <width>102</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Icon ändern</string>
   </property>
   <property name="text">
    <string>Icon ändern</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>66</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="toolTip">
       <string>Logtext-Erkennung hinzufügen</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonRemove">
      <property name="toolTip">
       <string>Logtext-Erkennung entfernen</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>81</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string>background-color: qlineargradient(spread:pad, x1:0.079, y1:0.0796818, x2:1, y2:0, stop:0 rgba(255, 255, 234, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Hier kann gesteuert werden, welche Icons einem bestimmten Logeintrag zugeordnet werden sollen. Das Suchmuster kann direkt in der Liste bearbeitet werden. Mehrere Suchmuster können durch ein Komma getrennt eingegeben werden. Ein manueller Eintrag ist im Allgemeinen nur erforderlich wenn für einen bestimmten Logeintrag kein Logbutton existiert. Beispielsweise für den variablen Logeintrag &quot;bin bei ...&quot;.</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LogIdentifierView</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LogIdentifierView</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
